<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ButtonsControlWindow</class>
 <widget class="QMainWindow" name="ButtonsControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>按钮控件样例</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
    <property name="icon">
     <iconset resource="controlres.qrc">
      <normaloff>:/images/buttonClose.png</normaloff>:/images/buttonClose.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{/*默认显示*/
border:1px;
border-radius:35px;/*圆角弧度(为正方形边长一半时就是圆形)*/
background-color:rgba(0,255,0,255);/*背景色*/
}
QPushButton:hover{/*鼠标悬停*/
background-color:rgba(0,0,255,255);
}
QPushButton:pressed{/*鼠标按下*/
background-color:rgba(255,0,0,255);
}</string>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>20</y>
      <width>70</width>
      <height>70</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>111</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>131</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton {                                                         
        border-radius: 10px;
        border: none;
        padding-left :10px;
        color: rgb(255,255,255);
        background: rgb(56,167,222); 
}
QToolButton:hover {                                                              

        padding-bottom: 2px;
        background: rgb(0,205,252); 
}

QToolButton:checked,QToolButton:pressed {      
        background: rgb(6,144,175); 
}</string>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testBtn1">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>170</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Test1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testBtn2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Test2</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox
{
    spacing: 2px;
    color: black;
}
 
QCheckBox::indicator {
	Width:45px;
	Height:30px;
}
QCheckBox::indicator:unchecked
{
    image: url(:/images/CheckboxOff);
}
QCheckBox::indicator:unchecked :hover
{
	image: url(:/images/CheckboxOff);
}
QCheckBox::indicator:checked
{
    image: url(:/images/CheckboxOn);
}
QCheckBox::indicator:checked :hover
{
    image: url(:/images/CheckboxOn);
}</string>
    </property>
    <property name="text">
     <string>C1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>230</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox
{
    spacing: 2px;
    color: black;
}
 
QCheckBox::indicator {
	Width:45px;
	Height:30px;
}
QCheckBox::indicator:unchecked
{
    image: url(:/images/CheckboxOff);
}
QCheckBox::indicator:unchecked :hover
{
	image: url(:/images/CheckboxOff);
}
QCheckBox::indicator:checked
{
    image: url(:/images/CheckboxOn);
}
QCheckBox::indicator:checked :hover
{
    image: url(:/images/CheckboxOn);
}</string>
    </property>
    <property name="text">
     <string>C2</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox
{
    spacing: 2px;
    color: black;
}
 
QCheckBox::indicator {
	Width:45px;
	Height:30px;
}
QCheckBox::indicator:unchecked
{
    image: url(:/images/CheckboxOff);
}
QCheckBox::indicator:unchecked :hover
{
	image: url(:/images/CheckboxOff);
}
QCheckBox::indicator:checked
{
    image: url(:/images/CheckboxOn);
}
QCheckBox::indicator:checked :hover
{
    image: url(:/images/CheckboxOn);
}</string>
    </property>
    <property name="text">
     <string>C3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="testBtn3">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>230</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Test3</string>
    </property>
   </widget>
   <widget class="QDialogButtonBox" name="buttonBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>330</y>
      <width>161</width>
      <height>32</height>
     </rect>
    </property>
    <property name="standardButtons">
     <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandLinkButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>CommandLinkButton</string>
    </property>
    <property name="description">
     <string>这是一个CommandLink按钮</string>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>170</y>
      <width>171</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButton1">
       <property name="text">
        <string>R1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton2">
       <property name="text">
        <string>R2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton3">
       <property name="text">
        <string>R3</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>151</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="radioButton4">
       <property name="styleSheet">
        <string notr="true">QRadioButton{
    spacing: 2px;
    color: black;
}
QRadioButton::indicator {
    width: 45px;
    height: 30px;
}
QRadioButton::indicator:unchecked {
    image: url(:/images/switchOff);
}
QRadioButton::indicator:unchecked:hover {
    image: url(:/images/switchOff);
}
QRadioButton::indicator:checked {
    image: url(:/images/switchOn);
}QRadioButton::indicator:checked:hover {
    image: url(:/images/switchOn);
}

</string>
       </property>
       <property name="text">
        <string>男</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton5">
       <property name="styleSheet">
        <string notr="true">QRadioButton{
    spacing: 2px;
    color: black;
}
QRadioButton::indicator {
    width: 45px;
    height: 30px;
}
QRadioButton::indicator:unchecked {
    image: url(:/images/switchOff);
}
QRadioButton::indicator:checked {
    image: url(:/images/switchOn);
}</string>
       </property>
       <property name="text">
        <string>女</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>536</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="controlres.qrc"/>
 </resources>
 <connections/>
</ui>
